frontend/src/entities/configuration/allowed-instance-types-configuration.tsx

```diff
@@ -0,0 +1,120 @@
+/**
+ Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+
+ Licensed under the Apache License, Version 2.0 (the "License").
+ You may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+
+ http://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+ */
+
+import {
+    Container,
+    ExpandableSection,
+    Header
+} from '@cloudscape-design/components';
+import React, { useMemo } from 'react';
+import { InstanceTypeMultiSelector } from '../../shared/metadata/instance-type-dropdown';
+
+export type AllowedInstanceTypesConfigurationProps = {
+    setFields: (any) => void;
+    expandedSections: {object};
+    setExpandedSections: (any) => void;
+    enabledNotebookInstanceTypes: string[];
+    enabledTrainingInstanceTypes;
+    enabledTransformInstanceTypes;
+    enabledEndpointInstanceTypes;
+};
+
+export function AllowedInstanceTypesConfiguration (props: AllowedInstanceTypesConfigurationProps) {
+    // Initialize the notebook instance type selectors with the currently selected options
+    const selectedNotebookInstanceOptions = useMemo(() => props.enabledNotebookInstanceTypes.map((instance) => {
+        return {
+            value: instance,
+            label: instance,
+        };
+    }), [props.enabledNotebookInstanceTypes]);
+
+    // Initialize the training job instance type selectors with the currently selected options
+    const selectedTrainingJobInstanceOptions = useMemo(() => props.enabledTrainingInstanceTypes.map((instance) => {
+        return {
+            value: instance,
+            label: instance,
+        };
+    }), [props.enabledTrainingInstanceTypes]);
+
+    // Initialize the transform job instance type selectors with the currently selected options
+    const selectedTransformJobInstanceOptions = useMemo(() => props.enabledTransformInstanceTypes.map((instance) => {
+        return {
+            value: instance,
+            label: instance,
+        };
+    }), [props.enabledTransformInstanceTypes]);
+
+    // Initialize the endpoint instance type selectors with the currently selected options
+    const selectedEndpointInstanceOptions = useMemo(() => props.enabledEndpointInstanceTypes.map((instance) => {
+        return {
+            value: instance,
+            label: instance,
+        };
+    }), [props.enabledEndpointInstanceTypes]);
+
+    return (
+        <>
+            <Container
```

Comment:
Is there a point to using `<>` when `<Container />` is the only child?

Reply:
There were talks about removing some of these containers, the empty tags don't really add much so didn't think it was a big deal having them here. I will remove.

Reply:
It isn't a big deal, we've just been trying to not have them when not necessary.